src/AppBundle/Utils/Markdown.php

```diff
@@ -23,15 +23,19 @@ class Markdown
 {
     private $parser;
 
+    private $purifier;
+
     public function __construct()
     {
         $this->parser = new \Parsedown();
+        $this->purifier = new \HTMLPurifier();
```

Comment:
Before doing that, are we sure we'll use the Purifier service anywhere else in the application?